tests/unit/tree_example

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
I try to use the data from the documentation, e.g., [this data](https://developer.github.com/v3/git/trees/#get-a-tree). This will necessitate updating the URL you have in the unit test file though. Cheers!